### PR TITLE
metrics: fix some panels of rc metrics that cannot show in the cluster deployed by the operator 

### PR DESCRIPTION
**Pull request by nolouch**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/56547

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
image
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: pkg/metrics/grafana/generate_json.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 go install github.com/google/go-jsonnet/cmd/jsonnet@latest
-git clone https://github.com/grafana/grafonnet-lib.git
+git clone https://github.com/nolouch/grafonnet-lib.git
```
Comment (glorv) [inline pkg/metrics/grafana/generate_json.sh:17]: Can you explain what does this fork fix? Why only resource control page is impacted？ Will we depend on this fork forever?

Comment (nolouch) [reply pkg/metrics/grafana/generate_json.sh:22]: update it in comments

**Review by glorv — APPROVED** [review]

**Review by crazycs520 — APPROVED** [review]

**Pull request merged**